com.unity.ml-agents/Tests/Runtime/RuntimeAPITest.cs

```diff
@@ -75,7 +75,7 @@ public IEnumerator RuntimeApiTestWithEnumeratorPasses()
 
             // Can't actually create an Agent with InferenceOnly and no model, so change back
             behaviorParams.BehaviorType = BehaviorType.Default;
-#if MLA_UNITY_PHSYICS_MODULE
+#if MLA_UNITY_PHYSICS_MODULE
```

Comment:
barely noticed the difference... not sure why this wasn't causing any trouble

Reply:
Accidentally found a bug in test - this was never run and in fact the test will fail if this part was enabled.

Reply:
whoops, thanks.